File: src/backend/EmployeeService/Models/Employee.cs
namespace EmployeeService.Models;

public class Employee
{
    public Guid Id { get; set; }
    public string Name { get; set; }
    public string Lastname { get; set; }
    public string PhoneNumber { get; set; }
    public Guid PositionId { get; set; }
    public Position Position { get; set; }
}

File: src/frontend/client-app/src/app/core/entities/employee.ts
export interface Employee
{
    id: string
    name?: string
    lastname?: string
    phoneNumber?: string
    positionId: string
}

export interface EmployeeCreate
{
    name?: string
    lastname?: string
    phoneNumber?: string
    positionId: string
}

export interface EmployeeUpdate
{
    id: string
    name?: string
    lastname?: string
    phoneNumber?: string
    positionId: string
}